desc/magnetic_fields/_core.py

```diff
@@ -1485,11 +1494,13 @@ def field_line_integrate(
     rtol=1e-8,
     atol=1e-8,
     maxstep=1000,
+    min_step_size=1e-8,
```

Comment:
I tried a few different values for this and it didn't seem to have a major effect on speed, and accuracy is fine as long as its reasonably small. We could do something fancy using the data from Albanese 2015 but I think probably not worth it, and if the user wants that they can still give a particular value for this or even give a different step size controller.